EIPS/eip-5555.md

```diff
@@ -0,0 +1,348 @@
+---
+eip: 5555
```

Comment:
```suggestion
eip: [to be assigned]
```
Numbers are assigned sequentially by editors and associates, you don't get to pick your own.